rllib/evaluation/rollout_worker.py

```diff
@@ -490,13 +490,14 @@ def make_env(vector_index):
                 rollout_fragment_length=rollout_fragment_length,
                 callbacks=self.callbacks,
                 horizon=episode_horizon,
-                pack_multiple_episodes_in_batch=pack,
+                multiple_episodes_in_batch=pack,
```

Comment:
Shortened this due to upcoming type-specs (will not fit into one line anymore after adding the type as well).